src/UKHO.ERPFacade.Common/Logging/EventIds.cs

```diff
@@ -17,6 +17,34 @@ public enum EventIds
         /// <summary>
         /// 920004 - New ENC content published event received by ERP Facade webhook.
         /// </summary>
-        NewEncContentPublishedEventReceived = 920004
+        NewEncContentPublishedEventReceived = 920004,
```

Comment:
For time being please use EventId start range as 940001 please as the used one already used in other project.

Reply:
Done